docs/api-docs/api-requirements.md

```diff
@@ -0,0 +1,265 @@
+        Supports use case:
+        <ul>
+            <li>
+                <a href="api-use-cases.md#cohesive-view"> Dashboard - One page view of specimen + library + sequencing artifacts </a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+    <tr>
+    <td class="tg-lboi">3</td>
+    <td class="tg-lboi">Resources are searchable by BICAN identifiers</td>
+    <td class="tg-lboi">
+        A user should be able to search resources by BICAN identifiers. Ideally, BICAN identifiers would be directly resolvable to resource (via a PURL system). The API should support retrieving full metadata for a resource with the BICAN identifier.
+    </td>
+    <td class="tg-lboi">Should have</td>
+    <td class="tg-lboi">
+        Supports use case:
+        <ul>
+            <li>
+                <a href="api-use-cases.md#cohesive-view"> Dashboard - One page view of specimen + library + sequencing artifacts </a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="tg-lboi">4</td>
+    <td class="tg-lboi">Response schemas include status where applicable</td>
+    <td class="tg-lboi">
+        A user should be able to query the API and receive status for a resource where applicable.
+    </td>
+    <td class="tg-lboi">Must have</td>
+    <td class="tg-lboi">
+        Supports use cases:
+        <ul>
+            <li>
+                <a href="api-use-cases.md#specimen-order-summary"> Dashboard - Specimen order status summary </a>
+            </li>
+            <li>
+                <a href="api-use-cases.md#sequencing-order-summary"> Dashboard - Sequencing order status summary </a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="tg-lboi">5</td>
+    <td class="tg-lboi">Programmatic update of controlled vocabulary</td>
+    <td class="tg-lboi">
+        A service account should be able to update controlled vocabulary terms from a centralized repo.
+    </td>
+    <td class="tg-lboi">Should have</td>
+    <td class="tg-lboi">
+        Supports use case: 
+        <ul>
+            <li>
+                <a href="api-use-cases.md#vocabulary-updates"> Controlled Vocabulary updates </a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <td class="tg-lboi">6</td>
+    <td class="tg-lboi">Programmatic update of biosample metadata</td>
+    <td class="tg-lboi">
+        A service account should be able to register biosample metadata with specimen portal to update specimen hierarchy
+    </td>
+    <td class="tg-lboi">Should have</td>
+    <td class="tg-lboi">
+        Supports use case: 
+        <ul>
+            <li>
+                <a href="api-use-cases.md#biosample-registration"> Biosample registration </a>
+            </li>
+        </ul>
+    </td>
+  </tr>
+</tbody>
+</table>
+
+## Examples
+
+### BICAN identifiers
+---
+
+
+```json
+GET https://example.com/api/donor/donor_1
+
+{
+    "id": "donor_1",
+    "bican_id": "BICAN:donor_1",
+    "age": ">89",
+    "specimen": [
```

Comment:
Use case to consider: The API consumer only wants top level donor information like "age" and "bican_id", but doens't need to know all the specimens associated with it. This API call could be slow if it was a REST API because it could return thousands of specimens in this array. A GraphQL interface wouldn't have a problem with this use case because users could just request the top level information without querying the whole list of specimens.

Reply:
See comment on graphql